src/com/lftechnology/java/training/alina/basic/stemleaf/DataSet.java

```diff
@@ -0,0 +1,32 @@
+package com.lftechnology.java.training.alina.basic.stemleaf;
+
+/**
+ * Class DataSet consists of data points to display stem and leaf
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class DataSet implements Comparable<DataSet> {
+	public int dataSet;
+
+	public DataSet(int dataSet) {
+		this.dataSet = dataSet;
+	}
+
+	public int getDataSet() {
+		return dataSet;
+	}
+
+	public void setDataSet(int dataSet) {
+		this.dataSet = dataSet;
+	}
+
+	@Override
+	public int compareTo(DataSet data) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Override "equals(Object obj)" to comply with the contract of the "compareTo(T o)" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1210)